ste/xfer-deleteBlob.go

```diff
@@ -14,20 +14,29 @@ import (
 
 var explainedSkippedRemoveOnce sync.Once
 
-func DeleteBlobPrologue(jptm IJobPartTransferMgr, p pipeline.Pipeline, pacer pacer) {
-
-	info := jptm.Info()
-	// Get the source blob url of blob to delete
-	u, _ := url.Parse(info.Source)
-
-	srcBlobURL := azblob.NewBlobURL(*u, p)
+func DeleteBlob(jptm IJobPartTransferMgr, p pipeline.Pipeline, pacer pacer) {
 
 	// If the transfer was cancelled, then reporting transfer as done and increasing the bytestransferred by the size of the source.
 	if jptm.WasCanceled() {
```

Comment:
Not really sure what you meant with this comment. Clarify?

Reply:
(er, this apparently is older code)

Reply:
Yeah, old code. Probably a copy and paste error in the comment (copied from a non-deletion case).  Will leave as is.